docs/tutorials/dotnet-systemd.md

```diff
@@ -92,55 +106,56 @@ First, get the IP address of your machine by running `ipconfig` in a PowerShell
 
 ![|624x357](assets/dotnet-systemd/ipconfig.png)
 
-Then select the setting icon in the corner of the Bot Framework Emulator and enter your IP under ‘localhost override'.
+Then select the setting icon in the bottom-left corner of the Bot Framework Emulator and enter your IP under ‘localhost override'.
 
 ![|624x411](assets/dotnet-systemd/emulator-settings.png)
 
-Click save and navigate back to the Welcome tab.
+Click 'Save' and navigate back to the Welcome tab.
 
 Click ‘Open Bot’ and under ‘Bot URL’ input:
-```
+
+```text
 http://localhost:3978/api/messages
 ```
 
 ![|624x411](assets/dotnet-systemd/open-a-bot.png)
 
-And click ‘Connect’ to connect to your Echo Bot running in WSL and start chatting!![|624x411](assets/dotnet-systemd/start-chatting.png)
+And click ‘Connect’ to connect to your Echo Bot running in WSL and start chatting!
+
+![|624x411](assets/dotnet-systemd/start-chatting.png)
 
 Congratulations, your Echo Chat Bot App is running on Ubuntu WSL as an App. Now it is time to make it run as a service.
 
 ## Running your Echo Bot as a systemd service
 
 Return to your running WSL distro and end the app with `Ctrl+C`.
 
-Then install the .Net systemd extension with:
+Then install the .NET systemd extension with:
 
->`$ sudo dotnet add package Microsoft.Extensions.Hosting.Systemd`
+> `sudo dotnet add package Microsoft.Extensions.Hosting.Systemd`
 
-Now we need to open our ‘echoes’ project in VS Code by running:
+We can open our project with VS Code by running this command in the 'echoes' directory:
```

Comment:
VS Code should be mentioned in the requiments with maybe a link to the "VS Code on WSL" tutorial.

Reply:
Yes, although any editor can be used and no unique functionality of VS Code is needed here.
I tried to adjust the language to say that VS Code is not a "need" but an option.
This could be made more clear by saying:

We now must open the project in a text editor. Here we use VS Code, which has excellent integration with WSL [point to VS Code tutorial].